.github/workflows/ci.yml

```diff
@@ -27,7 +27,7 @@ jobs:
           - windows-latest
           - macOS-latest
         arch:
-          - x64
+          - 'default'
```

Comment:
GitHub CI switched to arm on macOS. 'default' will use whatever arch the underlying machine has.